sale_triple_discount/models/sale_order_line.py

```diff
@@ -91,8 +91,8 @@ def _get_triple_discount(self):
             discount_factor *= (100.0 - discount) / 100.0
         return 100.0 - (discount_factor * 100.0)
 
-    def _prepare_invoice_line(self):
-        res = super()._prepare_invoice_line()
+    def _prepare_invoice_line(self, **optional_values):
```

Comment:
``**kwargs`` is the standard for such parameter.

Reply:
done